File: Unidad3/C#/u3p6.cs
using System;

class program
{
    static void Main()
    {
        Console.Write("Ingrese un número entero: ");
        int numero = int.Parse(Console.ReadLine());

        if (numero % 2 == 0)
        {
            Console.WriteLine("Es un número par.");
        }
        else
        {
            Console.WriteLine("Es un número impar.");
        }
    }
}

File: Unidad3/Java/u3p6.java
package com.mycompany.Programa6_U3;
import java.util.Scanner;

public class u3p6 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Ingrese un numero entero: ");
        int numero = scanner.nextInt();

        if (numero % 2 == 0) {
            System.out.println("Es un numero par.");
        } else {
            System.out.println("Es un numero impar.");
        }
    }
}
